test/bigtest/tests/line-create-test.js

```diff
@@ -18,17 +18,17 @@ describe('Create POL', () => {
   let order = null;
   const lineEditPage = new LineEditPage();
   const ACCOUNTS = [{
-    account_no: 'TEST_ACCOUNT',
+    accountNo: 'TEST_ACCOUNT',
   }];
   const CURRENCIES = ['GBR', 'USD'];
   const SUBSCRIPTION_INTERVAL = '10';
 
   beforeEach(function () {
     vendor = this.server.create('vendor', {
```

Comment:
should we rename all `vendor` to `organization` ?

Reply:
IMO no, as orders use only vendor organizations